apps/web/app/api/cron/weekly-summary/components/CreateReminderNotificationBody.tsx

```diff
@@ -0,0 +1,34 @@
+import { NotificationFooter } from "@/app/api/cron/weekly-summary/components/NotificationFooter";
+import { TNotificationResponse } from "@/app/api/cron/weekly-summary/types";
+import { Container, Text } from "@react-email/components";
+
+import { EmailButton } from "@formbricks/lib/emails/EmailButton";
+
+interface CreateReminderNotificationBodyProps {
+  WEBAPP_URL: string;
+  notificationData: TNotificationResponse;
+}
```

Comment:
can we use camelCase naming in all the props instead of uppercase(to keep the props consistent).